From 2c80ae23905e16eb0e545d62cb5785faa0776cbd Mon Sep 17 00:00:00 2001
From: Stefan Kalscheuer <stefan@stklcode.de>
Date: Fri, 15 Aug 2025 14:54:18 +0200
Subject: [PATCH] arm64: dts: marvell: reorder ethernet aliases for ESPRESSObin
 Ultra

Reorder aliases, so "ethernet[1-5] = &switch0port[1-5]" are aligned
insead of overriding ethernet1 to the WAN port (5).

Signed-off-by: Stefan Kalscheuer <stefan@stklcode.de>
---
 .../dts/marvell/armada-3720-espressobin-ultra.dts    | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/arch/arm64/boot/dts/marvell/armada-3720-espressobin-ultra.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-espressobin-ultra.dts
@@ -16,12 +16,12 @@
 		     "marvell,armada3720", "marvell,armada3700";
 
 	aliases {
-		/* ethernet1 is WAN port */
-		ethernet1 = &switch0port5;
-		ethernet2 = &switch0port1;
-		ethernet3 = &switch0port2;
-		ethernet4 = &switch0port3;
-		ethernet5 = &switch0port4;
+		/* for dsa slave device */
+		ethernet1 = &switch0port1;
+		ethernet2 = &switch0port2;
+		ethernet3 = &switch0port3;
+		ethernet4 = &switch0port4;
+		ethernet5 = &switch0port5;
 	};
 
 	/delete-node/ regulator;
